src/scene/container/bounds/getGlobalBounds.ts

```diff
@@ -84,12 +84,11 @@ export function _getGlobalBounds(
     }
     else
     {
-        if ((target as Renderable).addBounds)
+        if ((target as Renderable).bounds)
```

Comment:
I think it may be better to do this change in a separate PR to reduce the amount of noise here and ensure we don't have any breaking changes